test/signWithMultiSig.js

```diff
@@ -59,10 +63,14 @@ const createMultiSigAccount = async () => {
         gas: 900000,
     }
 
-    return caver.klay.sendTransaction(accountUpdateObject)
+    try {
+        await caver.klay.sendTransaction(accountUpdateObject)
+    } catch (error) {
+        throw error
+    }
 }
 
-before(function(done) {
+before(async function() {
```

Comment:
How about changing only before function like below?

```
before(async function() {
   ...
   await createMultiSigAccount()
})
```

I tested test code wont stuck with this change. (without adding `try-catch`)

Reply:
@jimni1222 
Thanks for your feed back.
That would be great idea.

I'll push the code reflecting your feedback on my forked branch `fix/stuck-at-exception`